bot/models/migrations/001_up__create_db.sql

```diff
@@ -55,4 +55,50 @@ CREATE TABLE IF NOT EXISTS polls
     created_at  TIMESTAMP,
     channel_id  BIGINT,
     message_id  BIGINT PRIMARY KEY
-)
+);
+
+CREATE TABLE IF NOT EXISTS levels
+(
+    guild_id           BIGINT,
+    user_id            BIGINT,
+    total_xp           INT DEFAULT 0,
+    last_msg           TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT PK_user PRIMARY KEY(guild_id,user_id)
+);
+
+CREATE TABLE IF NOT EXISTS roles
+(
+    role_id     BIGINT PRIMARY KEY,
+    guild_id    BIGINT,
+    name        VARCHAR,
+    color       VARCHAR
+);
+
+CREATE TABLE IF NOT EXISTS persistent_roles
+(
+    id                 SERIAL PRIMARY KEY,
+    guild_id           BIGINT,
+    user_id            BIGINT,
+    role_id            BIGINT,
+    CONSTRAINT fk_roles FOREIGN KEY(role_id) REFERENCES roles(role_id),
+    CONSTRAINT unique_guild_user_role UNIQUE (guild_id, user_id, role_id)
+);
+
+
+CREATE TABLE IF NOT EXISTS ignored_channels
+(
+    id          SERIAL PRIMARY KEY,
+    guild_id    BIGINT,
+    channel_id  BIGINT,
+    CONSTRAINT unique_ignored_channel UNIQUE (guild_id, channel_id)
+
+);
+
+CREATE TABLE IF NOT EXISTS levelling_roles
+(
+    id          SERIAL PRIMARY KEY,
+    guild_id    BIGINT,
+    role_id     BIGINT,
+    level       INT,
+    CONSTRAINT fk_levelling_roles FOREIGN KEY(role_id) REFERENCES roles(role_id)
+);
```

Comment:
You are not supposed to create these in the same migration file as we already have...
When a new PR is done you need to create a new migration file

**All** tables should have a unique `id` column. Yes, this isn't on all tables now but we are in working on improving the bot. I also want us to implement snowflake ids, like we have in our api
Postgres also has a naming convention for constraints - {tablename}_{columnname(s)}_{suffix}
I also don't see python models defined for all of these tables?


Now comments per table.

`levels`
Rename table to `levelling_users`

`roles`
Rename table to `custom_roles`

`persistent_roles`
Sorry this might have been my naming, but I think `persisted_roles` would be a better name.

`ignored_channels` rename to `levelling_ignored_channels`